test/dump/simd-binary.txt

```diff
@@ -0,0 +1,378 @@
+    f64x2.max)
+
+  ;; f32x4 add
+  (func (export "f32x4_add_0") (result v128)
+    v128.const i32 0x80000000 0xffc00000 0x449a5000 0xc49a5000
+    v128.const i32 0x00000000 0x3f800000 0x3f800000 0x3fc00000
+    f32x4.add)
+
+  ;; f64x2 add
+  (func (export "f64x2_add_0") (result v128)
+    v128.const i32 0x00000000 0x3ff80000 0x00000000 0xfff80000
+    v128.const i32 0x00000000 0xc0934a00 0x00000000 0x3ff00000
+    f64x2.add)
+
+  ;; f32x4 sub
+  (func (export "f32x4_sub_0") (result v128)
+    v128.const i32 0x80000000 0xffc00000 0x449a5000 0xc49a5000
+    v128.const i32 0x00000000 0x3f800000 0x3f800000 0x3fc00000
+    f32x4.sub)
+
+  ;; f64x2 sub
+  (func (export "f64x2_sub_0") (result v128)
+    v128.const i32 0x00000000 0x3ff80000 0x00000000 0xfff80000
+    v128.const i32 0x00000000 0xc0934a00 0x00000000 0x3ff00000
+    f64x2.sub)
+
+  ;; f32x4 div
+  (func (export "f32x4_div_0") (result v128)
+    v128.const i32 0x80000000 0xffc00000 0x3fc00000 0xc0400000
+    v128.const i32 0x00000000 0x3f800000 0x3f800000 0x3fc00000
+    f32x4.div)
+
+  ;; f64x2 div
+  (func (export "f64x2_div_0") (result v128)
+    v128.const i32 0x00000000 0x3ff80000 0x00000000 0xc0080000
+    v128.const i32 0x00000000 0x3ff00000 0x00000000 0x3ff80000
+    f64x2.div)
+
+  ;; f32x4 mul
+  (func (export "f32x4_mul_0") (result v128)
+    v128.const i32 0x80000000 0xffc00000 0x3fc00000 0xc0400000
+    v128.const i32 0x00000000 0x3f800000 0x3f800000 0x3fc00000
+    f32x4.mul)
+
+  ;; f64x2 mul
+  (func (export "f64x2_mul_0") (result v128)
+    v128.const i32 0x00000000 0x3ff80000 0x00000000 0xc0080000
+    v128.const i32 0x00000000 0x3ff00000 0x00000000 0x3ff80000
+    f64x2.mul)
+)
+
+(;; STDOUT ;;;
+
+simd-binary.wasm:	file format wasm 0x1
+
+Code Disassembly:
+
+00029c <i8x16_add_0>:
+ 00029f: fd 02 01 00 00 00 02 00 00 00 03 | v128.const 0x00000001 0x00000002 0x00000003 0x00000004
+ 0002b1: fd 02 01 00 00 00 02 00 00 00 03 | v128.const 0x00000001 0x00000002 0x00000003 0x00000004
+ 0002c3: fd 57                      | i8x16.add
+ 0002c4: 0b                         | end
+0002c5 <i16x8_add_0>:
+ 0002c8: fd 02 01 00 00 00 02 00 00 00 03 | v128.const 0x00000001 0x00000002 0x00000003 0x00000004
+ 0002da: fd 02 01 00 00 00 02 00 00 00 03 | v128.const 0x00000001 0x00000002 0x00000003 0x00000004
+ 0002ec: fd 68                      | i16x8.add
+ 0002ed: 0b                         | end
+0002ee <i32x4_add_0>:
+ 0002f1: fd 02 01 00 00 00 02 00 00 00 03 | v128.const 0x00000001 0x00000002 0x00000003 0x00000004
+ 000303: fd 02 01 00 00 00 02 00 00 00 03 | v128.const 0x00000001 0x00000002 0x00000003 0x00000004
+ 000315: fd 79                      | i32x4.add
+ 000316: 0b                         | end
+000317 <i64x2_add_0>:
+ 00031a: fd 02 01 00 00 00 02 00 00 00 03 | v128.const 0x00000001 0x00000002 0x00000003 0x00000004
+ 00032c: fd 02 01 00 00 00 02 00 00 00 03 | v128.const 0x00000001 0x00000002 0x00000003 0x00000004
+ 00033f: fd 8a                   | i64x2.add
```

Comment:
Hm, what's up with the `i64x2.*` instructions all being misaligned? Looks like `f32x4` and `f64x2` are too...

Reply:
I think it's actually the v128.consts that are unaligned because the immediates push them to the right.